x/epochstorage/keeper/keeper.go

```diff
@@ -71,3 +72,16 @@ func (k *Keeper) AddFixationRegistry(fixationKey string, getParamFunction func(s
 func (k *Keeper) GetFixationRegistries() map[string]func(sdk.Context) any {
 	return k.fixationRegistries
 }
+
+func (k Keeper) BeginBlock(ctx sdk.Context) {
+	if k.IsEpochStart(ctx) {
+		// run functions that are supposed to run in epoch start
+		k.EpochStart(ctx)
+
+		// Notify world we have a new session
+
+		details := map[string]string{"height": fmt.Sprintf("%d", ctx.BlockHeight()), "description": "New Block Epoch Started"}
+		logger := k.Logger(ctx)
```

Comment:
you can embed "logger" directly in the next line.

Reply:
done